pkgs/development/python-modules/pytest-testmon/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs
+, lib
+, pythonPackages
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-testmon";
+  version = "0.9.16";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "df00594e55f8f8f826e0e345dc23863ebac066eb749f8229c515a0373669c5bb";
+  };
+
+  buildInputs = with pythonPackages; [ pytest coverage ];
+  checkPhase = ''
+    pytest test
+  '';
+  doCheck = false;
```

Comment:
```suggestion
```

Reply:
Thank you for your help! I'm very grateful.

Reply:
Oh no problem :smile: I'm here to help.